.github/CODEOWNERS

```diff
@@ -44,8 +44,9 @@
 
 # Editor
 
+/editor/                                      @godotengine/docks
 /editor/script/                               @godotengine/script-editor
-/editor/shader/                               @godotengine/shaders
+/editor/shader/                               @godotengine/script-editor @godotengine/shaders
```

Comment:
Added this as I'd say the shader editor things also fall under the general editor UI part that's the responsibility of the script-editor team